server.js

```diff
@@ -358,59 +296,29 @@ io.on('connection', (socket) => {
                 top: topThreePolicies
             });
             socket.leave('sh-chancellor');
+            chanPolicy = data;
+            botPowers();
+            printResults();
         });
 
         socket.on('next-round', function (data) {
-            votersForGov = [];
-            votersAgainstGov = [];
-            if (data == '') {
-                nextPresident();
-                turnNum++;
-                io.emit('new-round', {
-                    presNom: presNom,
-                    pres: president,
-                    chan: chancellor,
-                    u: undesirables
-                });
-                undesirables.splice(0, 2);
-            } else {
-                turnNum++;
-                presNom = data;
-                io.emit('new-round', {
-                    presNom: data,
-                    pres: president,
-                    chan: chancellor,
-                    u: undesirables
-                });
-                undesirables.splice(0, 2);
-            }
+            nextRound(data);
         });
 
         socket.on('sh-player-killed', function (data) {
-            if (shPlayers.includes(data)) {
-                shPlayers.splice(shPlayers.indexOf(data), 1);
-            }
-            if (readyPlayers.includes(data)) {
-                readyPlayers.splice(readyPlayers.indexOf(data), 1);
-            }
-            if (liberals.includes(data)) {
-                liberals.splice(liberals.indexOf(data), 1);
-            }
-            if (fascists.includes(data)) {
-                fascists.splice(fascists.indexOf(data), 1);
-            }
-
-            io.emit('player-killed', data);
+            killPlayer(data);
         });
 
         socket.on('liberals-win', function (data) {
             lWins++;
             io.emit('sh-game-finished', data);
+            gameOver = true;
         });
 
         socket.on('fascists-win', function (data) {
             fWins++;
             io.emit('sh-game-finished', data);
+            gameOver = true;
```

Comment:
added the gameOver variable to prevent bot algorithm to continue after the game has ended